### PR TITLE
Add user & group difference

### PR DESCRIPTION
**Pull request by nkt**

### CODE REVIEW

File: proto/peers.proto [hunk 1]
```diff
@@ -47,3 +47,25 @@ message GroupOutPeer {
     int32 group_id = 1 [(dlg).log="visible"];
     int64 access_hash = 2 [(dlg).log="danger"];
 }
+
+message UserOutPeerInTime {
```
Comment (ostronom) [inline proto/peers.proto]: Why do we need additional entities when we have UserOutPeer/GroupOutPeer already? Adding int64 field is both-ways compatible afaik.

Comment (dmtryyy) [reply proto/peers.proto]: There could be just *OutPeer field and clock in this models.

File: proto/sequence_and_updates.proto [hunk 2]
```diff
@@ -267,11 +267,24 @@ message ResponseGetReferencedEntitites {
...
 }
 
+message RequestGetEntitiesDifference {
+    option (scalapb.message).extends = "im.dlg.grpc.GrpcRequest";
+    repeated UserOutPeerInTime users = 1;
+    repeated GroupOutPeerInTime groups = 2;
+    repeated GroupMembersSubsetInTime group_members = 3;
+}
+
+message ResponseGetEntitiesDifference {
```
Comment (dmtryyy) [inline proto/sequence_and_updates.proto]: Don't you need current clocks?

File: proto/peers.proto (hunk 1 above)
Comment (ostronom) [reply proto/peers.proto]: I think we should stick to Occam and his razorblade.

**Pull request closed without merging**